assets/icons/close.svg

```diff
@@ -1,10 +1,5 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_1081_3634)">
-<path d="M25.3333 8.54667L23.4533 6.66667L16 14.12L8.54663 6.66667L6.66663 8.54667L14.12 16L6.66663 23.4533L8.54663 25.3333L16 17.88L23.4533 25.3333L25.3333 23.4533L17.88 16L25.3333 8.54667Z" fill="#585858"/>
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why did you decrease the size of this icon? Did you check that it still looks as intended in all usages?

Reply:
seems like this happened during merging. no clue why. gonna use the close social with different color and check the occurrences